### ISSUE TITLE
Header ImageView not hidden sometimes and drawn below the listview 

### ISSUE THREAD

**Issue by Nammari** [ManuelPeinado/FadingActionBar#18]
I am using your library in my project but seems that there is a bug . 

I have a listview in a fragment and for the header image i load it from url . now every thing works great 
but sometimes while scrolling the imageview in the header does not hide and drawn  below the listview . 

please fix this .

**Comment by ManuelPeinado** [comment]
Hi, this is a known issue and I don't have a fix for it (sorry). However, it doesn't manifest itself simply by scrolling, you have to either:

a) change the orientation of the screen.
b) call nofifyDataSetChanged() on your adapter.

This is only happen for ListViews, ScrollViews work fine. So maybe if your content is not too long you could consider putting it in a vertical LinearLayout that's inside a ScrollView. Other than that there's not much I can tell you, I'll try to find a fix for this issue over the Christmas vacations but I've tried and failed in the past so there are no guarantees I will succeed this time.

**Comment by ghost** [comment]
Do you have found a fix for this problem?
Without it i can't use this..

And if set the transparency of the Actionbar to transparent, the whole view gets destroyed and shown like very weird..

01-26 18:19:41.001: W/View(10377): requestLayout() improperly called by android.widget.GridView{6593e2c8 VFED.VC. ......ID 1080,0-2160,16777041 #7f0600af app:id/dategrid} during layout: running second layout pass
01-26 18:19:41.001: W/View(10377): requestLayout() improperly called by android.widget.FrameLayout{65673208 V.E..... ......ID 0,0-1080,16777214} during layout: running second layout pass
01-26 18:19:41.001: W/View(10377): requestLayout() improperly called by android.view.View{656509d8 V.ED.... ......ID 0,16777214-1080,16779134 #7f0600b9 app:id/fab__listview_background} during layout: running second layout pass

If the layout displays messed up, that's what I get. If everything works properly i get

01-26 18:20:44.963: W/View(10377): requestLayout() improperly called by android.widget.FrameLayout{65a48c40 V.E..... ......ID 0,0-1080,16777214} during layout: running second layout pass
01-26 18:20:44.963: W/View(10377): requestLayout() improperly called by android.view.View{65a48748 V.ED.... ......ID 0,16777214-1080,16779134 #7f0600b9 app:id/fab__listview_background} during layout: running second layout pass

**Comment by dalexanco** [comment]
Yep, that's weird, sometime (seems to be random) I have a white line on top of my header and this same message in log "requestLayout() improperly called by android.widget.FrameLayout (...)  during layout: running second layout pass"
